package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aurelia-cli",
-  "version": "0.32.0",
+  "version": "0.33.0",
```

Comment:
could you undo this change? Our release process updates this automatically

Reply:
sure. one sec